src/components/authentication/LoginModal.vue

```diff
@@ -0,0 +1,88 @@
+<script setup>
+import GenericToast from '../generics/GenericToast.vue'
+import { VueFinalModal } from 'vue-final-modal'
+import { ref, computed } from 'vue'
+import LoginForm from './LoginForm.vue'
+import RegisterForm from './RegisterForm.vue'
+import { useVfm } from 'vue-final-modal'
+
+const vfm = useVfm()
+
+//Form Management
+
+const modalType = ref(false) //False - Login, True - Register
+
+function changeModal() {
+  modalType.value = !modalType.value
+}
+
+//Toast management
+
+const loginSuccess = ref(false)
+const loginFinished = ref(false)
+const toastType = computed(() => {
+  return loginSuccess.value ? 'success' : 'error'
+})
+const toastMessage = computed(() => {
+  return loginSuccess.value ? 'Login Successful' : 'Login Failed'
+})
+
+function finishLoginSuccess() {
+  console.log('Login done')
+  loginSuccess.value = true
+  loginFinished.value = true
+  //Needed to trigger Toast. The toast is not inside the Modal, but its appearance depends on loginFinished becoming true,
```

Comment:
Too long comment. Please refer to this information on Trello in your task or in commit/pr description

Reply:
Nevermind, the comment is used for worning others about this line of code!